docker-compose.yaml

```diff
@@ -28,7 +28,7 @@ services:
       AZURE_STORAGE_ACCOUNT_NAME: ${AZURE_STORAGE_ACCOUNT_NAME:-devstoreaccount1}
       AZURE_STORAGE_USE_CONNECTION_STRING: "true"
       STATEMENT_VERSION: v1
-      STATEMENT_RECEIVER_ENDPOINT: http://localhost:3000
+      STATEMENT_RECEIVER_ENDPOINT: http://localhost:3021
```

Comment:
For local development, receiver is bound to localhost port `3021` so updating so link is usable locally.